swarm-private/values.yaml

```diff
@@ -20,6 +20,19 @@ bootnode:
     publicaddr: 846c424961adc146d54861bdf1eb6015e6908b689fd12d01c61307fffc848c22e514f5c898dc9243fbb17aa80750b556772599d84fe86a4b715f40ebc4c049bf
     nodekey: 09a28f7be03b782768ae2a8e2de90e7e27cd643a60f2a68e57514097bfa23566
     extraFlags: []
+  persistence:
+    ## this enables PVC templates that will create one per pod
+    enabled: false
+    ## swarm data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClass: "-"
+    accessMode: ReadWriteOnce
+    size: 1Gi
```

Comment:
A bit big EBS, but I guess there aren't any smaller `gp2` EBS volumes.

Reply:
Correct. Minimum for `gp2` is `1Gi`